app/controllers/admin/source_files_controller.rb

```diff
@@ -13,6 +13,13 @@ def after_resource_updated_path(resource)
     end
 
     def filter_resources(resources, search_term:)
+      resources =
+        if Administrate::SourceFileSearch.user_is_filtering_by_status?(search_term)
```

Comment:
I would have preferred to have this conditional up in `scoped_resource` above, but that was a little tricky since we don't have access to the `search_term` param. I tried just using `params["search"]` but I guess administrate does some processing to the raw param value before passing it in here, so it seemed simpler to just adjust this method. I don't think this is super weird though since the method is called `filter_resources` and we are in fact filtering resources 😅